src/cart/src/include/cart/swim.h

```diff
@@ -1,40 +1,24 @@
-/* Copyright (c) 2016 UChicago Argonne, LLC
- * Copyright (C) 2018-2019 Intel Corporation
- * All rights reserved.
+/*
```

Comment:
we need to keep the UChicago stuff copyright

Reply:
Done